Makefile.am

```diff
@@ -251,6 +251,7 @@ include src/static/Makefile-static.am
 include src/remotectl/Makefile-remotectl.am
 include src/reauthorize/Makefile-reauthorize.am
 include src/selinux/Makefile-selinux.am
+include src/auto-ssh-agent/Makefile-auto-agent.am
```

Comment:
Suggest just calling this Makefile.am. The longer names are a hold over from old code.